integration/feature/dynamic-cross-module/repo/build.sc

```diff
@@ -1,10 +0,0 @@
-import mill._, scalalib._
```

Comment:
Deleted this guy because it's basically a duplicate of `example/misc/5-dynamic-cross-module/`

Reply:
Are examples also run as integration test?